### ISSUE TITLE
Remove incubator flamechart views from trace server

### ISSUE THREAD

**Issue by bhufmann** [eclipse-tracecompass-incubator/org.eclipse.tracecompass.incubator#12]
With the migration of the incubator flame chart and related views as well as their implementation to mainline Trace Compass https://github.com/eclipse-tracecompass/org.eclipse.tracecompass/pull/9 the trace server will now show 3 different flame chart views (classic, incubator and new) in the available views. As a first step the incubator flamechart plug-in should be removed from the trace server.

Removing the flamechart plug-in will require other incubator plug-ins to remove their dependencies to the incubator flamechart plug-in and use the mainline implementation instead.

### Plug-ins to remove from traceserver.product
- org.eclipse.tracecompass.incubator.callstack.core
- org.eclipse.tracecompass.incubator.analysis.core

### Plug-ins to update
- org.eclipse.tracecompass.incubator.rocm.core
-  org.eclipse.tracecompass.incubator.traceevent.core
- org.eclipse.tracecompass.incubator.traceevent.core.tests

Note: The source code migrated to the mainline Trace Compass should be removed from the incubator repository.

**Comment by arfio** [comment]
I agree that this should be done after the following pull request is merged: 
https://github.com/eclipse-tracecompass/org.eclipse.tracecompass/pull/23
Aside from that: 
- I have updated the ROCm core plugin, I need to push the changes and make a PR for it.
- The other plugins should follow.

**Comment by bhufmann** [comment]
I noticed that `org.eclipse.tracecompass.incubator.callstack.core.sampled.callgraph` package hasn't been migrated to mainline. This needs to be done before removing the incubator plug-in.

**Comment by arfio** [comment]
The `org.eclipse.tracecompass.incubator.callstack.core.sampled.callgraph` package was added to this PR: https://github.com/eclipse-tracecompass/org.eclipse.tracecompass/pull/23
